src/ExternalImage.php

```diff
@@ -50,57 +50,62 @@ class ExternalImage implements ImageInterface
      * @api
      * @var string
      */
-    private $alt_text;
+    protected string $alt_text;
 
     /**
-     * Caption.
+     * Alt text.
      *
      * @api
      * @var string
      */
-    private $caption = '';
+    protected string $caption;
+
+    /**
+     * Representation.
+     *
+     * @var string What does this class represent in WordPress terms?
+     */
+    public static $representation = 'attachment';
 
     /**
      * File.
      *
      * @api
-     * @var mixed
+     * @var string
      */
-    public $file;
+    protected string $file;
 
     /**
      * File location.
      *
      * @api
      * @var string The absolute path to the attachmend file in the filesystem
-     *             (Example: `/var/www/htdocs/wp-content/themes/my-theme/images/my-pic.jpg`)
+     *             (Example: `/var/www/htdocs/wp-content/uploads/2015/08/my-pic.jpg`)
```

Comment:
I would keep the `/themes/my-theme/images/` path, because that’s a typical location where an ExternalImage might live.